js_modules/dagster-ui/packages/ui-core/src/settings/SettingsMainPane.tsx

```diff
@@ -17,7 +17,7 @@ export const SettingsMainPane = () => {
         <Route path="/deployment/daemons">
           <InstanceHealthPageContent />
         </Route>
-        <Route path="/deployment/concurrency">
+        <Route path="/deployment/concurrency(/?.*)">
```

Comment:
Could this be `/deployment/concurrency/:concurrencyKey?`? Or is the `concurrencyKey` value too complex for this?

Reply:
I wanted the URL to match both the top-level `/deployment/concurrency` url as well as the detailed pool-specific `/deployment/concurrency/foo` url.

Reply:
I was thinking that using `:concurrencyKey?` (with the `?`) would allow this, but off the top of my head I don't remember whether that's the way to do that with react-router.

Reply:
oh, you're right, that does work